### PR TITLE
[FIX] account: inv. report price_average

### PR DESCRIPTION
**Pull request by mhabboush**
When converting a price (average price in this case) the price should be multiplied by the source uom factor (uom_line.factor)
https://github.com/odoo/odoo/blob/72fea2fe11244a55e9a630049fc75f147e26e473/addons/uom/models/uom_uom.py#L172
follow-up fix to 978012b38abb941c88fe85f1befa2ac659545915

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mhabboush** [comment]
@william-andre

**Review by bfr-o — APPROVED** [review]
LGTM. Maybe @smetl can have a look as he had a look at the previous PR.

**Review by william-andre — CHANGES_REQUESTED** [review]
Please change the division of a division of a division of a multiplication of a division.
Use parenthesis, the order of operations can be hard to read as they depend on the language/formalism.

**Comment by mhabboush** [comment]
> Please change the division of a division of a division of a multiplication of a division.
> Use parenthesis, the order of operations can be hard to read as they depend on the language/formalism.

better now?

File: addons/account/report/account_invoice_report.py [hunk 1]
```diff
@@ -98,10 +98,11 @@ def _select(self):
                 line.quantity / NULLIF(COALESCE(uom_line.factor, 1) / COALESCE(uom_template.factor, 1), 0.0) * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
                                                                             AS quantity,
                 -line.balance * currency_table.rate                         AS price_subtotal,
-                -COALESCE(line.balance
-                   / NULLIF(line.quantity, 0.0)
-                   / NULLIF(COALESCE(uom_line.factor, 1), 0.0)
-                   / NULLIF(COALESCE(uom_template.factor, 1), 0.0),
+                -COALESCE(
+                   # Average line price
```
Comment (william-andre) [inline addons/account/report/account_invoice_report.py:102]: Comments in SQL are `-- inline comment` or `/* block comment */`

**Comment by mhabboush** [comment]
@william-andre

**Review by william-andre — APPROVED** [review]
Better indeed, thanks!

@robodoo r+

**Comment by william-andre** [comment]
@robodoo rebase-merge

**Comment by william-andre** [comment]
@robodoo rebase-ff

**Pull request closed without merging**